CAIPs/caip-261.md

```diff
@@ -13,67 +13,82 @@ requires (*optional):
 ---
 
 ## Simple Summary
-<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the CAIP.-->
-CAIP-x defines a way to assess trust in software components leveraging community claims and social relationships of trust.
+
+CAIP-x defines a way to assess trust in discreet resources (e.g. software components or packages) by leveraging community claims and pulling in trust data from social graphs and webs of trust.
 
 ## Abstract
-<!--A short (~200 word) description of the technical issue being addressed.-->
-This proposal introduces a standardized data framework with the aim to standardize:
-- the claims made by communities (peers) about software components,
-- the claims made by communities about peers,
-- the community-derived trust scores of software components and/or peers.
 
-These data elements can be used independently. 
+This proposal introduces a standardized data framework with the aim to propose lightweight interfaces for:
+
+1. claims made by actors in an open, peer-to-peer network about discreet resources (software components in all the examples that follow),
+2. claims made by these actors about each other,
+3. community-derived trust scores for the discreet resources and/or actors.
+
+These three data elements can be used independently of one another.
 For building a community-powered trust assessment mechanism, it's necessary to identify and gather the relevant data in accordance with the requirements of the intended trust scores computation.
-These data can then be leveraged to effectively calculate trust scores, reflecting the overall sentiment of the communit.
+These data can then be leveraged to compute synthetic trust scores which reflect the overall sentiment of the community.
 
-The standardized data framework includes:
+The data framework modeled below incorporates the following basic primitives as inputs:
 
-- **Trust:** This allows individuals to define their trusted peers, shaping their trust graph;
-- **Security Report:** This enables security expert to publish security insights about software components;
-- **Review:** This allows individuals to endorse or dispute any claims as well as any entity such as software components;
-- **Trust score:** This enables any trust computer to publish computed trust scores about software components and/or peers.
+- **Trust List:** This allows individuals to define their trusted peers, shaping their trust graph;
+- **Expert Report:** This enables experts to publish insights and conclusions about a given resource (e.g. an audit in the case of software components);
```

Comment:
The "Expert Report" naming broadens the applicability of the assertion to encompass all types of expert reports, beyond just security. This versatility could be beneficial. Accordingly, we should update the name of the credential below to reflect this change.